app/_locales/en/messages.json

```diff
@@ -2446,10 +2458,10 @@
     "message": "WARNING: Never disclose your Secret Recovery Phrase. Anyone with this phrase can take your Ether forever."
   },
   "secretPhrase": {
-    "message": "Only the first account on this wallet will auto load. After completing this process, to add additional accounts, click the drop down menu, then select Create Account."
+    "message": "If you restore a wallet using the Secret Recovery Phrase, only the wallet’s initial account will appear at first. You will need to $1 created using the SRP. "
```

Comment:
I would like to double-check the existence of "SRP" since we spell this out in the rest of the copy and we don't indicate to the user that "SRP" stands for "Secret Recovery Phrase" elsewhere.

What if we make either of these two changes:
1. Use `Secrete Recovery Phrase (SRP)` earlier to use `SRP` in the rest of the copy
2. Replace `SRP` with `Secret Recovery Phrase` here

Reply:
Great catch @digiwand! I suggest this copy fix that also fixes a slightly wrong information we were giving on the previous sentence:
"If you restore a wallet using the Secret Recovery Phrase, only the wallet’s initial account might appear at first. You'll be able to [re-add any other accounts](https://metamask.zendesk.com/hc/en-us/articles/360015489271-How-to-add-missing-accounts-after-restoring-with-Secret-Recovery-Phrase) created previously."

Reply:
> Good catch @digiwand , My brain might have auto filled SRP when I read through the copy before 😅

I can say the same lol 😅